Java/src/main/java/planes/PassengerPlane.java

```diff
@@ -26,13 +26,6 @@ public String toString() {
                 '}');
```

Comment:
Please remove comments with fields (like this - `//=================FIELDS=================`) from this class or add to other classes for consistency

Reply:
Were removed in different classes